UpdateDependencies.ps1

```diff
@@ -16,7 +16,7 @@ param(
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
-    [string]$GitHubPullRequestNotifications='')
+    [string]$GitHubPullRequestNotifications='stephentoub;ericstj;ellismg;weshaggard;joshfree')
```

Comment:
Ok.  I can close this PR, but where is the official Subscriptions.json file?  I don't see one in corefx, nor do I see a versions repo under dotnet.